src/main/java/com/example/demo/Application.java

```diff
@@ -6,7 +6,6 @@
 @SpringBootApplication
 public class Application {
     public static void main(String[] args) {
-        //ApplicationContext context = new AnnotationConfigApplicationContext("com.example.demo.services");
```

Comment:
why aren't these changes made in a separate PR? This one is about greeting repository.